packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/__tests__/quanticStandaloneSearchBox.test.js

```diff
@@ -109,6 +119,107 @@ describe('c-quantic-standalone-search-box', () => {
   afterEach(() => {
     cleanup();
   });
+  describe('with default options', () => {
+    it('should create the component', async () => {
+      const element = createTestComponent();
+      await flushPromises();
+
+      expect(element).not.toBeNull();
+
+      const interfaceElem = element.shadowRoot.querySelector(
+        selectors.searchInterface
+      );
+      expect(interfaceElem).not.toBeNull();
+
+      const input = interfaceElem.querySelector(selectors.searchBoxInput);
+      expect(input).not.toBeNull();
+      expect(input.withoutSubmitButton).toBe(false);
```

Comment:
here you could probably iterate over the defaultOptions object instead of having so many expect...

Reply:
ahhh okok